### DISCUSSION TITLE
Tidy Tuesday Exercise for March 27, 2023 Lab Meeting

### DISCUSSION THREAD

**Discussion by valeste** [RobertsLab/resources#1613] [General]
Hi all,
For next weeks lab meeting we will be doing another Tidy Tuesday exercise!

**Step 1**: Go to the [Tidy Tuesday GitHub repo](https://github.com/rfordatascience/tidytuesday) and read the README.md

**Step 2**: This week will be working through the [Programming Languages](https://github.com/rfordatascience/tidytuesday/blob/master/data/2023/2023-03-21/readme.md) data set. 

**Step 3**: Follow the instructions to download the data set:
```{r}
# requirements
library(tidyverse)
# Get the Data

# Read in with tidytuesdayR package 
# Install from CRAN via: install.packages("tidytuesdayR")
# This loads the readme and all the datasets for the week of interest

# Option 1.1-1.2 : Either ISO-8601 date or year/week works!

 tuesdata <- tidytuesdayR::tt_load('2023-03-21')
 tuesdata <- tidytuesdayR::tt_load(2023, week = 12)

languages <- languages

# Option 2: Or read in the data manually

languages <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2023/2023-03-21/languages.csv')
```
I found that reading in the data manually using option 2 was easiest for me.

**Step 4**: Look through the data and come up with some questions that you think you may be able to answer!

**Step 5**: Wrangle that data and generate plots to answer the questions you came up with.

At the next lab meeting we will go around and share the questions we had, how we tackled them, issues we ran into, and plots! Feel free to post your workings down below. 

See y'all soon!

**Comment by AHuffmyer** [comment]
This was fun! Here is a knitted output with my script and plots! https://github.com/AHuffmyer/TidyTuesday/blob/main/21March2023/script.md

**Comment by valeste** [comment]
Here's my lab notebook [post](https://valeste.github.io/2023-03-22-Tidy-Programming-Languages/) for this week's Tidy Tuesday Exercise!

**Comment by sr320** [comment]
here is what I came up with 
https://github.com/sr320/nb-2023/blob/main/misc/code/02-tidy-tuesday.md
